app/uk/gov/hmrc/securemessage/scheduler/SchedulingConfig.scala

```diff
@@ -28,7 +30,7 @@ trait SchedulingConfig {
   def durationFromConfig(propertyKey: String): FiniteDuration =
     configuration
       .getOptional[FiniteDuration](s"scheduling.$name.$propertyKey")
-      .getOrElse(throw new IllegalStateException(s"$propertyKey missing"))
+      .getOrElse(FiniteDuration(0, TimeUnit.MILLISECONDS))
```

Comment:
Is it better to throw an exception here?

Reply:
Basically this `getOrElse` itself  is not required as we have default values in the config itself. The reason for me to change it is thats scala3 is expecting an extra case at the calling method, which I believe is not required. Hence I changed it like this. But, feel free to suggest.. thanks